zulipterminal/model.py

```diff
@@ -289,7 +289,13 @@ def current_narrow_contains_message(self, message: Message) -> bool:
             # all messages contains all messages
             not self.narrow
             # mentions
-            or (self.narrow[0][1] == "mentioned" and "mentioned" in message["flags"])
+            or (
+                self.narrow[0][1] == "mentioned"
+                and (
+                    "mentioned" in message["flags"]
+                    or "wildcard_mentioned" in message["flags"]
```

Comment:
I'm unsure if we should use the same set intersection here, but it perhaps points towards using that internally for simplicity (and slight optimization) via a refactor.